src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageItemControl.xaml

```diff
@@ -537,17 +539,17 @@
                   <Setter TargetName="_updateButton" Property="Visibility" Value="Collapsed" />
                 </DataTrigger>
 
-                <!-- The installed version indicator becomes invisible when
+                <!-- The deprecation indicator becomes invisible when
```

Comment:
Is this statement really correct? :)

Reply:
What...does it do?

I just replaced the `_installedIndicator` with the `_deprecationIndicator` because we wanted the same behavior.

Reply:
I'd  imagine it's opposite. 

It becomes visible if IsPackageDeprecate = true.